### ISSUE TITLE
Deal With Spaces When Converting

### ISSUE THREAD

**Issue by KaspianDev** [supsm/MCPPPP#22]
**Describe the bug**
When convering in pack that has spaces in folder names, minecraft crashes, because converter errors during convertion.

**Screenshots**

**Important stuff**
- OS: Windows
- Version/Commit Number: 0.6.1

**Resourcepack**
https://www.curseforge.com/minecraft/texture-packs/bettervanillabuilding

**Logs**
```
[20:56:18] Error: tried to copy nonexistent file
mcpppp-temp/BetterVanillaBuildingV2.71/assets/minecraft/textures/optifine/cit/items/christmas/string lights/tree/yellowtree.png
[20:56:18] Error: tried to copy nonexistent file
mcpppp-temp/BetterVanillaBuildingV2.71/assets/minecraft/textures/optifine/cit/items/christmas/string lights/tree/yellowtree.png
```

**Comment by supsm** [comment]
Can you send full logs? It may be more useful

**Comment by supsm** [comment]
Note to self: the crash is actually caused by an assertion in reselect's add_random. It doesn't show up in the logs however, since assertions are only outputted to standard output. I'll try to see if I can get around this somehow

**Comment by supsm** [comment]
The crash should be fixed in the latest commit

**Comment by KaspianDev** [comment]
Same result

**Comment by supsm** [comment]
Should be fixed in 0.7.0, sorry for the late notice
